src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        this.color = color;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return firstLeg * secondLeg * 0.5;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, color: " + getColor() + ", area: "
+                + getArea() + " sq. units, first leg: " + firstLeg + " units, second leg: "
+                + secondLeg + " units");
```

Comment:
When constructing the string in the `draw` method, consider using `String.format` for better readability. Also, make sure to use `name()` method instead of `getColor()` when referring to enum constants for a string representation, if `color` is an enum type.